package.json

```diff
@@ -77,7 +77,10 @@
     "auto-changelog": "^2.4.0",
     "eslint": "^8.28.0",
     "node-fetch": "^2.6.0",
-    "semver": "^7.3.5"
+    "postcss-less": "^6.0.0",
```

Comment:
Please also add postcss@^8.3.5 because action ci log says so
```
warning " > postcss-less@6.0.0" has unmet peer dependency "postcss@^8.3.5".
```

Reply:
👍, fixed in https://github.com/fomantic/Fomantic-UI/pull/2592/commits/7614896e14bd6c69cf9789b56896646fb92c91d5